### PR TITLE
Histology

### PR DESCRIPTION
**Pull request by oliche**
We reviewed the models Thursday however this is only the tip of the iceberg.

This is the full app with:
- models
- admin interface
- REST endpoints
- REST filters
- REST tests (no logic to test in models)

@rossant the choice field is a strange beast when it comes to REST filtering/serializing

I'm already using those endpoints in the histology pipeline.

### CODE REVIEW

**Comment by oliche** [comment]
Update instructions

```
./manage.py makemigrations  # this should return "no changes detected"
./manage.py migrate
./manage.py loaddata ./experiments/fixtures/experiments.coordinatesystem.json
./manage.py loaddata ./experiments/fixtures/experiments.probemodel.json
./manage.py set_user_permissions
```

File: alyx/alyx/urls.py [hunk 1]
```diff
@@ -34,6 +34,8 @@
 urlpatterns = [
     path('', mv.api_root),
 
+    path('', include('experiments.urls')),
```
Comment (rossant) [inline alyx/alyx/urls.py]: is the '' normal here?

Comment (oliche) [reply alyx/alyx/urls.py]: yup I decided to keep the urls in the app itself for easier refactoring. 
Do you think it will cause issues ?

Comment (rossant) [reply alyx/alyx/urls.py]: I meant the first argument to `path()` -- but actually I see that the URLs are properly named in `alyx/experiments/urls.py` so we're good!

**Comment by oliche** [comment]
Live demo: https://dev.alyx.internationalbrainlab.org/admin/experiments/trajectoryestimate/

**Pull request merged**